docs/compare.md

```diff
@@ -8,6 +8,8 @@ After a load test is complete, a report can be generated and saved to a file whi
 go run ./cmd/ltctl report generate --output=base.out --label=base "2020-06-23 07:23:35" "2020-06-23 07:33:35"
 ```
 
+Note that in the case of a local deployment, you'll need to set the `--prometheus-url ` flag to the URL of the Prometheus server from where the tool will scrape the results.
```

Comment:
this is great, but I think there is a need to add a link that points to how to set this up in order to collect the data in the prometheus server.

Reply:
Yes..

Reply:
Check the latest commit https://github.com/mattermost/mattermost-load-test-ng/pull/698/commits/f9b28abb33f7a8d4b31a8d8158609e557921c2aa and let me know if that's what you had in mind!

Reply:
This is great!